koku/masu/processor/ocp/ocp_cost_model_cost_updater.py

```diff
@@ -50,11 +50,6 @@ def __init__(self, schema, provider):
             self._tag_default_supplementary_rates = cost_model_accessor.tag_default_supplementary_rates
             self._distribution = cost_model_accessor.distribution
 
-    @property
-    def is_amortized(self):
```

Comment:
Any particular reason this public property was removed without a deprecation period and now the private attribute is being used directly?

This public property isn't doing much today, but allows how the value is determined to change in the future without breaking any calling code. It also provides a way to cache the value on the object if it is expensive to calculate.

Reply:
It was introduced in the previous PR as a hacky way to get unit tests to run. We fixed that, but forgot to remove the property until this PR. I love properties, but we don't need a depreciation period here. Nothing else uses this and we don't have 3rd parties using things where we need a period like that. The goal is to remove this check entirely but for now this let's us do both paths based on the unleash flag while we're getting this feature change out. You'd also need a cached_property to cache, otherwise it's calculated every time.